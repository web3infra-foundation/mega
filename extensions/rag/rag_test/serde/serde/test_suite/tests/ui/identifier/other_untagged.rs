use serde_derive::Deserialize;

#[derive(Deserialize)]
#[serde(untagged)]
enum F {
    #[serde(other)]
    Other,
}

fn main() {}
