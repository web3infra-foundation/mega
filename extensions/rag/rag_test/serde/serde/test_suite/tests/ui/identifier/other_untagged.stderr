error: #[serde(other)] cannot appear on untagged enum
 --> tests/ui/identifier/other_untagged.rs:6:5
  |
6 | /     #[serde(other)]
7 | |     Other,
  | |_________^
